### PR TITLE
New issue templates

### PR DESCRIPTION
**Pull request by jmklix**
*Issue #, if available:*

*Description of changes:*
use new github forms for issue templates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    attributes:
+      label: Version
+      description: |
+        List the versions & crates of the aws-rust-sdk you are using.
+
+        `cargo install cargo-tree`
+        (see install here: https://github.com/sfackler/cargo-tree)
+
+        Then:
+        `cargo tree | grep aws-sdk-`
```
Comment (jdisanti) [inline .github/ISSUE_TEMPLATE/---bug-report.yml:24]: We have crates without `sdk-` in the prefix. This should be:
```suggestion
        `cargo tree | grep aws-`
```

File: .github/ISSUE_TEMPLATE/---bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        if you haven't already (e.g., `tracing_subscriber::fmt::init();`),
+        and then setting the environment variable `RUST_LOG` before
+        running your program, as follows:
+        `RUST_LOG='smithy_http_tower::dispatch=trace,smithy_http::middleware=trace'`
+
+        For example:
+        `RUST_LOG='smithy_http_tower::dispatch=trace,smithy_http::middleware=trace' cargo run`
+
+        The SDK redacts sensitive information such as auth headers in these trace logs,
+        but please look through them before posting just to be sure.
```
Comment (jdisanti) [inline .github/ISSUE_TEMPLATE/---bug-report.yml:73]: Can this be its own textarea?

File: .github/ISSUE_TEMPLATE/---questions-help.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      options:
+        - label: I've searched for [previous similar issues](https://github.com/awslabs/aws-sdk-rust/issues) or [discussions](https://github.com/awslabs/aws-sdk-rust/discussions) and didn't find any solution.
+          required: true
+        - label: I searched the [Developer Guide](https://docs.aws.amazon.com/sdk-for-rust/latest/dg/welcome.html) for an answer to my question.
+          required: true
+  
+  - type: input
+    id: issue
+    attributes:
+      label: General Issue
```
Comment (jdisanti) [inline .github/ISSUE_TEMPLATE/---questions-help.yml:20]: Why are there two question fields?

**Review by jdisanti — COMMENTED** [review]
Looks mostly good, just some minor things.

File: .github/ISSUE_TEMPLATE/---bug-report.yml (hunk 2 above)
Comment (jmklix) [reply .github/ISSUE_TEMPLATE/---bug-report.yml:78]: added new logs textarea

**Review by jdisanti — APPROVED** [review]

**Pull request merged**